src/components/connection-health/connection-health.css

```diff
@@ -1,7 +1,7 @@
-
-.connection-wrapper{
-    padding-top: 70px;
+.connection-wrapper {
+    padding-right: 70px;
```

Comment:
use rem instead

Reply:
In other branch I will change any place with px to rem